### ISSUE TITLE
Use keys of different length in let & map alignment example

### ISSUE THREAD

**Issue by clyfe** [bbatsov/clojure-style-guide#219]
https://guide.clojure.style/#bindings-alignment
https://guide.clojure.style/#map-keys-alignment

:thing1 & :thing2 have the same length.
Use keys of different length in let & map alignment example, to make obvious that it's the keys to be aligned, not values.

```clojure
{:thing1 thing1
 :long-thing long-thing}
```

**Comment by bbatsov** [comment]
Fair point. We might also mention that it's find align the values as well, if one would like to make them stand out a bit more.
